src/Phinx/Console/Command/Create.php

```diff
@@ -79,6 +83,63 @@ protected function getCreateMigrationDirectoryQuestion()
     }
 
     /**
+     * Get the question that allows the user to select which migration path to use.
+     *
+     * @param string[] $paths
+     * @return ChoiceQuestion
+     */
+    protected function getSelectMigrationPathQuestion(array $paths)
+    {
+        return new ChoiceQuestion('Which migrations path would you like to use?', $paths, 0);
+    }
+
+    /**
+     * Returns the migration path to create the migration in.
+     * 
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return mixed
+     * @throws \Exception
+     */
+    protected function getMigrationPath(InputInterface $input, OutputInterface $output)
+    {
+        // First, try the non-interactive option:
+        $path = $input->getOption('path');
+
+        if (!empty($path)) {
+            return $path;
+        }
+
+        $paths = $this->getConfig()->getMigrationPaths();
+
+        // No paths? That's a problem.
+        if (empty($paths)) {
+            throw new \Exception('No migration paths set in your Phinx configuration file.');
+        }
+
+        $paths = Util::globAll($paths);
+
+        if (empty($paths)) {
+            throw new \Exception(
+                'You probably used curly braces to define migration path in your Phinx configuration file, ' .
+                'but no directories have been matched using this pattern. ' .
+                'You need to create a migration directory manually.'
+            );
+        }
+
+        // Only one path set, so select that:
+        if (1 === count($paths)) {
+            return array_shift($paths);
```

Comment:
Only if the indicies are not numeric or ordered. I'm happy with what it is at the moment.